starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +90,22 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray (wordsUnique){
+  for ( i = 0; i < wordsUnique.length; i++){
+    for ( j = i+1 ; j < wordsUnique.length - 1; j++){
+      if (wordsUnique[i] == wordsUnique[j]){
+        uniquifyArray.splice (j,j);
```

Comment:
How splice works is:
`splice(index, howMuch)` 
- index: array's position
- howMuch: how much elements you want remove

So you should have `uniquifyArray.splice (j,1);`

But this is dangoures since you are changing the `.length` of the array. So the loop will fail in some point. 
We saw how to do it today. If you have question, ask us.